### ISSUE TITLE
twisted/python/filepath.py has several lines wider than 79 columns

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7537]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#7537|
|Type|defect|
|Created|2014-07-10 07:32:35Z|
|Branch|https://github.com/twisted/twisted/tree/cleanup-7537|

I end up looking at this file a lot, it would be nice to fix up some of the coding-standard violations.

<details><summary>Searchable metadata</summary>

```
trac-id__7537 7537
type__defect defect
reporter__glyph glyph
priority__normal normal
milestone__ 
branch__branches_cleanup_7537 branches/cleanup-7537
branch_author__glyph glyph
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1404977555948707 1404977555948707
changetime__1405019877616735 1405019877616735
version__None None
owner__glyph glyph

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph commented|
|-|-|

(In [42854]) Branching to cleanup-7537.

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|Alex commented|
|-|-|

Looks good to me, please land.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set status to `closed`|
|-|-|

(In [42856]) Merge cleanup-7537

Author: glyph

Reviewer: Alex

Fixes: [#7537](https://github.com/twisted/twisted/issues/7537)

Correct some minor coding-standard violations in filepath, re-wrapping
lines longer than 79 columns.
